requirement.txt

```diff
@@ -1,13 +1,25 @@
-jcconv
-simplejson
-Flask
-Flask-Babel
-Flask-cors
-pyusb>=1.0.0b2
-pyserial
-pif
-Pillow
-netifaces
-pycups
-# temporary desactivate because of a xmlescpos pip issue
-#xmlescpos
+Babel==2.2.0
+Flask==0.10.1
+Flask-Babel==0.9
+Flask-Cors==2.1.2
+itsdangerous==0.24
+jcconv==0.2.3
+Jinja2==2.8
+MarkupSafe==0.23
+netifaces==0.10.4
+pif==0.7.3
+Pillow==3.2.0
+pycountry==1.20
+# Temporary disable for support windows
+#pycups==1.9.73
+pyserial==3.0.1
+pytz==2016.3
+pyusb==1.0.0rc1
+git+https://github.com/OpenPymeMx/py-xml-escpos.git
+requests==2.9.1
+simplejson==3.8.2
+six==1.10.0
+speaklater==1.3
+Unidecode==0.4.19
+Werkzeug==0.11.5
+wheel==0.24.0
```

Comment:
Maybe you are right @legalsylvain I create a virtualenv but installed also the PyPostelium library, so its probably some of the added dependencies are from pypostelium, going to check and update the MR acording